superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -144,9 +151,25 @@ const RightMenu = ({
       {menu.label}
     </>
   );
+
+  const handleMenuSelection = (itemChose: any) => {
```

Comment:
Is `itemChose` a complicated type or would this be able to be defined with more specific than `any`?